community/acknowledgements.html

```diff
@@ -46,9 +46,10 @@ <h2>Technical Resources</h2>
 
                 <dd>
                   <p>Hosting of the Boost mailing lists, our web server, our
-                  <a href="http://svn.boost.org/trac/boost">bug tracker</a>,
-                  is donated by the <a href="http://www.osl.iu.edu/" class=
-                  "external">Open Systems Lab at Indiana University</a>.
+                  <a href="http://svn.boost.org/trac/boost">wiki and old
+                  bug tracker</a>, is donated by the 
+                  <a href="http://www.osl.iu.edu/" class= "external">Open
+                  Systems Lab at Indiana University</a>.
```

Comment:
Perhaps this could just be merged, then @mjcaisse could make update to the part about the current infrastructure sponsor, as we discussed on cpplang.slack.com.